### PR TITLE
Add Logz.io provider

### PR DESCRIPTION
**Pull request by pengux**
### Example

```
LOGZIO_API_TOKEN=foobar LOGZIO_BASE_URL=https://api-eu.logz.io ./terraformer import logzio -r=alerts,alert_notification_endpoints // Import Logz.io alerts and alert notification endpoints
```

### Provider
logzio provider >=1.1.1 - [here](https://github.com/jonboydell/logzio_terraform_provider/)

### List of supported Logz.io resources:

 * `alerts`
    * `logzio_alert`
* `alert notification endpoint`
    * `logzio_endpoint`

### CODE REVIEW

**Comment by islomar** [comment]
@googlebot I consent.

File: README.md [hunk 1]
```diff
@@ -503,6 +504,23 @@ List of supported Cloudflare services:
   * `cloudflare_zone`
   * `cloudflare_record`
 
+
+### Use with Logz.io
+
+Example:
+
+```
+ LOGZIO_API_TOKEN=foobar LOGZIO_BASE_URL=https://api-eu.logz.io ./terraformer import logzio -r=alerts,alert_notification_endpoints // Import Logz.io alerts and alert notification endpoints
```
Comment (sergeylanzman) [inline README.md]: You can add flag api-token and base-url. See example in github provider. not must...

File: README.md [hunk 2]
```diff
@@ -503,6 +504,23 @@ List of supported Cloudflare services:
...
+```
+
+List of supported Logz.io resources:
+
+* `alerts`
+    * `logzio_alert`
+* `alert notification endpoint`
+    * `logzio_endpoint`
```
Comment (sergeylanzman) [inline README.md]: I see provider support users too. Any plans add users to terraformer?

File: providers/logzio/alerts.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+// Generate Terraform Resources from Logzio API,
+func (g *AlertsGenerator) InitResources() error {
+	var client *alerts.AlertsClient
+	client, _ = alerts.New(g.Args["token"].(string), g.Args["baseURL"].(string))
+
+	alerts, err := client.ListAlerts()
+	if err != nil {
+		log.Println(err)
```
Comment (sergeylanzman) [inline providers/logzio/alerts.go]: you don't need log.println here. Error return from func and printed in main flow

File: providers/logzio/alert_notification_endpoints.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+// Generate Terraform Resources from Logzio API,
+func (g *AlertNotificationEndpointsGenerator) InitResources() error {
+	var client *endpoints.EndpointsClient
+	client, _ = endpoints.New(g.Args["token"].(string), g.Args["baseURL"].(string))
+
+	endpoints, err := client.ListEndpoints()
+	if err != nil {
+		log.Println(err)
```
Comment (sergeylanzman) [inline providers/logzio/alert_notification_endpoints.go]: you don't need log.println here. Error return from func and printed in main flow

File: providers/logzio/logzio_provider.go [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+var (
+	disallowedChars = regexp.MustCompile(`[^A-Za-z0-9-]`)
+)
+
+const logzioProviderVersion = "~>v1.1.1"
+
+func (p LogzioProvider) GetResourceConnections() map[string]map[string][]string {
+	return map[string]map[string][]string{}
```
Comment (sergeylanzman) [inline providers/logzio/logzio_provider.go]: You can connect alert and endpoint here.
See example in GCP and AWS provider or other

File: terraform_utils/provider_wrapper/provider.go [hunk 6]
```diff
@@ -146,7 +146,7 @@ func (p *ProviderWrapper) initProvider() error {
 			AllowedProtocols: []plugin.Protocol{plugin.ProtocolGRPC, plugin.ProtocolNetRPC},
 			Logger: hclog.New(&hclog.LoggerOptions{
 				Name:   "plugin",
-				Level:  hclog.Info,
```
Comment (sergeylanzman) [inline terraform_utils/provider_wrapper/provider.go]: return to info

File: README.md (hunk 1 above)
Comment (pengux) [reply README.md]: We tried with this approach first, but it didn't work as the provider also needs the same env vars set. So we removed the flags for those arguments and instead just use the env vars.

File: README.md (hunk 2 above)
Comment (pengux) [reply README.md]: Yes, we haven't tested that yet as the user resource support is relatively new in the provider. Will add it some time in the future.

File: providers/logzio/logzio_provider.go (hunk 5 above)
Comment (islomar) [reply providers/logzio/logzio_provider.go]: We have tried to connect it, but it doesn't work for us: https://github.com/form3tech-oss/terraformer/commit/ca8f4427027b893c5ebe3279ee3db59bee25b17e

Do you know what we are doing wrong?

File: providers/logzio/alert_notification_endpoints.go (hunk 4 above)
Comment (islomar) [reply providers/logzio/alert_notification_endpoints.go]: Fixed.

File: terraform_utils/provider_wrapper/provider.go (hunk 6 above)
Comment (islomar) [reply terraform_utils/provider_wrapper/provider.go]: Fixed.

File: providers/logzio/logzio_provider.go [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+var (
+	disallowedChars = regexp.MustCompile(`[^A-Za-z0-9-]`)
+)
+
+const logzioProviderVersion = "~>v1.1.1"
+
+func (p LogzioProvider) GetResourceConnections() map[string]map[string][]string {
+	return map[string]map[string][]string{
+		"alert": {"endpoint": []string{"alert_notification_endpoints", "id"}},
```
Comment (sergeylanzman) [inline providers/logzio/logzio_provider.go]: change to => `"alerts": {"alert_notification_endpoints": []string{"alert_notification_endpoints", "id"}},`

File: providers/logzio/alerts.go (hunk 3 above)
Comment (islomar) [reply providers/logzio/alerts.go]: Fixed.

File: providers/logzio/logzio_provider.go (hunk 7 above)
Comment (islomar) [reply providers/logzio/logzio_provider.go]: Done, thank you very much! 🙏

**Comment by sergeylanzman** [comment]
Please rebase to one commit and resolve conflict. I want release new version.
Thank you!

**Comment by pengux** [comment]
@sergeylanzman Done. Thanks!

**Pull request merged**